### ISSUE TITLE
Covers missing alt text.

### ISSUE THREAD

**Issue by rkingett** [mouse-reeve/fedireads#153]
I'm a screen reader user and, on the demo site at least, I've noticed that none of the books have alt tags on the covers, which can make it problematic for a screen reader user to browse the most recently added books and more, including shelves.

I'm a screen reader user that uses the NVDA screen reader.

Could you make this standard? Have the alt text say something like, "BOOK TITLE COVER." or just "BOOK TITLE." and make it standard for all added or imported books?

**Comment by mouse-reeve** [comment]
should be fixed! thanks for pointing this out

**Comment by rkingett** [comment]
Yup! It's fixed. Thanks for making this the default for anybody that wants to use this. Now they will have to do less work to make their websites accessible. Would you know of anyone hosting this as a stable version yet?

**Comment by rkingett** [comment]
While looking at the [other formats page,](https://fedireads-test.glitch.me/editions/47254013) I can't tell, using NVDA, what format a book is. Is there some way to add that as text next to the image or make it so that the format variable is included in those alt texts?

**Comment by mouse-reeve** [comment]
good call about the other edition, that page needs a lot of work. 

there isn't a stable instance at this point, I don't think the code is quite there yet.

**Comment by mouse-reeve** [comment]
@rkingett  I expanded the cover alt text to include the format and the publication year, as available. I think it clarifies the difference between the covers on the edition page, but is it too much information for the feed page? I can have a long version and a short version if so.

**Comment by rkingett** [comment]
I commented on the commit, but I actually just now realized I was moving through different editions of the book. Keep the alt text as is, with year included. I didn't know the other editions just had covers and not text. I really like what you've done wit the alt text. I think it will be very useful the way it is, but...

If you plan to include text in the other editions section, remove the links from the grapphics and shorten the alt text, since there will be text anyway.

If not, things are fine as they are. I just wasn't sure if I was looking through different editions.

Maybe make the edition results an ordered list? I'm not sure how messy that would look visually, but it would help in navigation, especially if a book doesn't have edition information.

1. Less. Audio CD. 2018.
2. Less. Audio CD. 2019.

Let me know, either here, or on the other commit.

**Comment by mouse-reeve** [comment]
Okay, I made the edition page on ordered list. The data isn't perfect, so for some books, the year or the format isn't available. It can be added manually in the edit book page. 

Here's an example with 50 different editions: https://fedireads-test.glitch.me/editions/47254166

I'm not sure I follow what you're saying works best for the alt texts and links. When the cover isn't available, it used to show both the title and author and the alt text on the blank placeholder image. I changed it to set the alt tag to "no cover" and show the information in the text. Does that address the duplication, or are you seeing it in a different place?

**Comment by rkingett** [comment]
That fixed the tripple link issue! Now it is much much easier to skin and navigate by unvisited link and list item. This is great!

**Comment by mouse-reeve** [comment]
glad to hear it, thank you so much helping me with this!
